### ISSUE TITLE
UnicodeEncodeError

### ISSUE THREAD

**Issue by morpheus-87** [mattrobenolt/jinja2-cli#77]
Since today, I have an issue to use your great tool: I always get the following error, when I want to convert a template to HTML:
```
Traceback (most recent call last):
  File "/usr/local/bin/jinja2", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/jinja2cli/cli.py", line 335, in main
    sys.exit(cli(opts, args))
  File "/usr/local/lib/python2.7/dist-packages/jinja2cli/cli.py", line 261, in cli
    sys.stdout.write(output)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xfc' in position 436: ordinal not in range(128)
```
What could be the reason for this error? I haven't changed anything in the template.

**Comment by mattrobenolt** [comment]
What version are you using? And can you give me a test to reproduce? It's likely an easy fix.

**Comment by morpheus-87** [comment]
I'm using these versions:
```
jinja2-cli v0.6.0
 - Jinja2 v2.10.1
```

I think, I fixed it with an upgrade to version 0.7.0 :)

**Comment by mattrobenolt** [comment]
Ah, yeah, I assumed it was good with latest version. :)
